controllers/auth.controller.js

```diff
@@ -91,6 +92,7 @@ module.exports.signup = (req, res) => {
 
                     user.save()
                         .then((user) => {
+                            Logger.addLog('authentication',`${req.userData.firstname} ${req.userData.lastname} 'created an account'`);
```

Comment:
I didn't think about this before, but `req.userData` will only exist if the user has a token. When a user is signing up, there is no way they can have a token because they don't have an account to log in with yet.

Luckily for us, the signup route body contains all the useful information we will need. Instead of writing `req.userData.firstname` and `req.userData.lastname`, use `req.body.firstname` and `req.body.lastname`.

Another thing, let's remove those quotes around 'create an account' so that these particular log items look consistent with all of the others.